### PR TITLE
TRIT-59: Service Landing Page

### PR DESCRIPTION
**Pull request by apodi**
TRIT-59: Service Landing Page

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -129,3 +129,19 @@ compliant.nine.p2.1=The service was built using parts that were tested by the
...
+homePage.p2.2=available to you, including:
+homePage.numberOfDays.l1=number of days you were present in the UK
+homePage.numberOfDays.l2=number of days you were working in the UK
+homePage.warning=HMRC will not be bound by the results where the information you provide does not accurately reflect your facts and circumstances.
+homePage.helpAndGuidance.h2=Help and guidance
+homePage.p3=Your residence status can only be decided by reference to established facts and circumstances.
+homePage.p4.1=It is recommended that you read the parts of the guidance in the
+homePage.p4.anchor=Statutory Residence Test (SRT)
+homePage.p4.2=that are relevant to your circumstances before you start to help you answer the questions accurately.
+homePage.p5=The results given rely on you providing accurate information. You will be able to print a copy of the result and keep this for future reference.
```
Comment (awasnik) [inline conf/messages.en]: add additional blank line

File: conf/views/homePage.njk [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          }) }}
+
+          <h2 class="govuk-heading-m">{{ messages("homePage.helpAndGuidance.h2") }}</h2>
+
+          <p class="govuk-body">{{ messages("homePage.p3") }}</p>
+
+          <p class="govuk-body">{{ messages("homePage.p4.1") }} <a class="govuk-link" target="_blank" href="{{ config.statutoryResidenceTestUrl }}">{{ messages("homePage.p4.anchor") }}</a>
+                    {{ messages("homePage.p4.2") }}</p>
+          <p class="govuk-body">{{ messages("homePage.p5") }}</p>
+
```
Comment (awasnik) [inline conf/views/homePage.njk]: there are two blank spaces, remove additional blank line.

File: conf/messages.en (hunk 1 above)
Comment (apodi) [reply conf/messages.en]: fixed

File: conf/views/homePage.njk (hunk 2 above)
Comment (apodi) [reply conf/views/homePage.njk]: fixed

**Pull request merged**